roles/kubernetes-apps/ansible/templates/dns-autoscaler.yml.j2

```diff
@@ -40,8 +40,9 @@ spec:
         kubernetes.io/os: linux
       tolerations:
         - effect: NoSchedule
-          operator: Equal
```

Comment:
The reason why this line is removed is the default value of operator is `Equal` and most other manifests don't specify it?

Reply:
Yup default value is `Equal` for operator